src/app/components/DotMenu.tsx

```diff
@@ -48,8 +48,7 @@ export const DotMenuDropdownList = styled(DropdownList)`
 `;
 
 // tslint:disable-next-line:variable-name
-// @ts-ignore
-export const DotMenuDropdown = styled((props: DropdownProps) => <Dropdown toggle={<DotMenuToggle />} {...props} />)`
+export const DotMenuDropdown = styled((props: DropdownProps) => <Dropdown {...{...props, toggle: <DotMenuToggle />}} />)`
```

Comment:
TS indicated that `props` would always overwrite `toggle`, so I rewrote to ensure `toggle` has precedence.

Reply:
I believe you should be able to just move the toggle property to before the props property. And maybe also change the type of props to `Omit<DropdownProps, 'toggle'>`

Sorry, I started looking into this because it shows up as assigned to me in the GitHub UI but eventually I saw it was actually Josiah so I'll let him do the rest :)

Reply:
yea you should just be able to spread props before the extra param
```suggestion
export const DotMenuDropdown = styled((props: DropdownProps) =>  <Dropdown {...props} toggle={<DotMenuToggle />} />)`
```

i believe this issue is happening because of a bug in the styled-component prop types where custom styled component props like this one wouldn't get strong type checks (so we were actually not providing a toggle prop even if its required by the types)